zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/exception/CorruptedNoteException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.notebook.exception;
+
+import java.io.IOException;
+
+public class CorruptedNoteException extends IOException {
+    public CorruptedNoteException(final String message, Exception e) {
```

Comment:
Better to add noteId in constructor

Reply:
Seems need to reconstruct the method `fromJson` to pass the parameter `noteId`, or maybe I can create another constructor for `CorruptedNoteException` with one more parameter `noteId` and add the noteId into it after catching the exception without noteId?
![image](https://user-images.githubusercontent.com/34905992/124433984-eb2efe80-dda5-11eb-95e8-c9c93f988d1e.png)

Reply:
noteId added